### ISSUE TITLE
Certain tests cause segfaults on `ROCArray`s

### ISSUE THREAD

**Issue by leios** [leios/Fae.jl#31]
The issue does not seem to be present on `Array` or `CuArray` and seems to be an issue in the following tests:

```
            layering_testsuite(ArrayType)
            chaos_testsuite(ArrayType)
            object_testsuite(ArrayType)
```

It could be because the gridsize is less than the number of threads being used? I thought that was taken care of in KA with the `ndrange` variable, but might be mistaken.

I am tracking this issue separately in case the segfault appears elsewhere.

Note: This *can* cause a system freeze if you are using the GPU for visualization as well.

**Comment by leios** [comment]
Check out `AMDGPU#jps/dev` and see if it is fixed

**Comment by leios** [comment]
This seems to be fixed as of #64
